src/main/kotlin/screens/ArchiveList.kt

```diff
@@ -0,0 +1,55 @@
+package screens
+
+import models.Archive
+
+class ArchiveList : Screen() {
+    private var list: MutableList<Archive> = mutableListOf<Archive>()
+
+    override fun showMenu() {
+        println("Меню:")
+        println("0. Создать архив")
+        println("1. Просмотр архивов")
+        println("2. Выход")
+    }
+
+    fun createArchive() {
+        println("Введите название архива:")
+
+        var name: String? = ""
+
+        while (name!!.isEmpty()) {
```

Comment:
Лучше не использовать оператор `!!`, ведь иначе программа крашнется
```suggestion
        while (name?.isEmpty() == true) {
```